Protocol.md

```diff
@@ -165,6 +163,7 @@ and closing transactions.
    * [`spk`:`change_spk`]
    * [`cet_adaptor_signatures`:`cet_adaptor_signatures`]
    * [`signature`:`refund_signature`]
+   * [`negotiation_fields`:`negotiation_fields`]
```

Comment:
Should the type definition be moved here? I feel it's becoming a bit unclear which types are defined here and which are defined in `Messaging.md`, do we have a clear rule for that?

Reply:
This is a good question, I don't have a good answer but someone should come up with one and open a PR to make things uniform. I've opened an issue (#141)

Reply:
Perfect